pulumi/backuprunner/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:18.12.1-alpine
+
+COPY . .
+
+RUN yarn build
+
+WORKDIR dist
+
+ENTRYPOINT ["/usr/local/bin/node", "index.js", "$@"]
```

Comment:
we can also rely on ci for our build artifacts and just copy it in:

```
COPY . .

ENTRYPOINT ["node", "dist/index.js"]
```
no need for `$@` entrypoint args passed in via docker run